package.json

```diff
@@ -277,6 +277,7 @@
     "emptykit.css": "1.0.1",
     "enzyme": "3.11.0",
     "enzyme-adapter-react-16": "1.15.6",
+    "ethereum-blockies": "git+https://github.com/jamesmudgett/blockies.git",
```

Comment:
i would suggest pinning to a specific commit or tag here. also some questions:

1.  can we move this repo from james' personal account to the brave github org?
2. how do we plan to keep this repo up-to-date with important fixes in upstream such as security fixes?

Reply:
Thank you for the suggestions! The repo has been moved here <https://github.com/brave/blockies> and is also now pinned to the latest commit. As far as keeping the repo up-to-date, this repo does not rely on any dependancies at all. It's basically just a JS function that returns value, so I don't believe that will be an issue. :)